test/system/irohad_test.cpp

```diff
@@ -328,12 +328,15 @@ TEST_F(IrohadTest, RestartWithOverwriteLedger) {
  * restart
  * @given an Iroha with some transactions commited ontop of the genesis
  * block
- * @when the Iroha is restarted without --overwrite-ledger flag
+ * @when the Iroha is restarted without ---blockstore_strategy=continue flag
  * @then the state is successfully restored
  *  AND the Iroha accepts and able to commit new transactions
  */
 TEST_F(IrohadTest, RestartWithoutResetting) {
-  launchIroha();
+  launchIroha(config_copy_,
+              path_genesis_.string(),
+              path_keypair_.string(),
+              std::string("--blockstore_strategy=continue"));
```

Comment:
Why do you need `continue` flag on the first run?

Please do not pass any additional flags for the first run.